src/site/stages/build/drupal/graphql/GetAllPages.graphql.js

```diff
@@ -9,7 +9,7 @@ const vamcOperatingStatusAndAlerts = require('./vamcOperatingStatusAndAlerts.gra
 const fragments = require('./fragments.graphql');
 const newsStoryPage = require('./newStoryPage.graphql');
 const sidebarQuery = require('./navigation-fragments/sidebar.nav.graphql');
-const alertsQuery = require('./alerts.graphql');
```

Comment:
Should we delete this `alerts.graphql` as part of this PR? looks like both the references to it are removed with this work.

Reply:
:+1: @rianfowler